nix/modules/joex.nix

```diff
@@ -178,8 +178,8 @@ let
 
       wkhtmlpdf = {
         command = {
-          program = "${pkgs.wkhtmltopdf}/bin/wkhtmltopdf";
```

Comment:
I think we should rather use the config option for weasyprint which has been added in the last release. I think I just missed to update the nix config values accordingly :( You can then switch to weasyprint by setting `html-converter = weasyprint` (see [defaults](https://docspell.org/docs/configure/defaults/#joex))

Reply:
Got it. One thing to note - if `pkgs.wkhtmltopdf` is at all mentioned in the joex config - nix will complain about the insecure package. So, I think the changes should be:

1. Add `html-converter`, default to `weasyprint`
2. Add `weasyprint` set of options, enable by default
3. Disable `wkhtmltopdf` by default, add some documentation note about the package being not secure

Reply:
Yes that sounds great! I think we can just leave wkhtmltopdf empty or so, users could still set it if they want to.